domain-models-interface-extensions/src/main/java/org/folio/rest/tools/GenerateRunner.java

```diff
@@ -99,30 +108,33 @@ public static void main(String[] args) throws Exception {
 
     String [] ramlFiles = System.getProperty("raml_files", SOURCES_DEFAULT).split(",");
     String pre = ramlFiles[0];
-    File x0 = new File(pre);
-    File x1 = x0;
+    File input = new File(pre);
+    File temp = input;
     while (true) {
-      x1 = x1.getParentFile();
-      if (x1 == null) {
+      temp = temp.getParentFile();
+      if (temp == null) {
         break;
       } else {
-        if (x1.getName().equals(SOURCES_DEFAULT)) {
-          x0 = x1;
+        if (temp.getName().equals(SOURCES_DEFAULT)) {
+          input = temp;
```

Comment:
Thank you for better variable names!
Can you do us a favor and move this file computation in a method so that only

    File input = rebase(ramlFiles[0]);

remains? (You may come up with some better method name.)

Reply:
Yeah.